docs/examples/hello_pt_job_api.rst

```diff
@@ -0,0 +1,273 @@
+.. _hello_pt_job_api:
+
+Hello PyTorch with Job API
+==========================
+
+Before You Start
+----------------
+
+Feel free to refer to the :doc:`detailed documentation <../programming_guide>` at any point
+to learn more about the specifics of `NVIDIA FLARE <https://pypi.org/project/nvflare/>`_.
+
+We recommend you first finish the :doc:`Hello FedAvg with NumPy <hello_fedavg_w_numpy>` exercise since it introduces the
+federated learning concepts of `NVIDIA FLARE <https://pypi.org/project/nvflare/>`_.
+
+Make sure you have an environment with NVIDIA FLARE installed.
+
+You can follow :ref:`getting_started` on the general concept of setting up a
+Python virtual environment (the recommended environment) and how to install NVIDIA FLARE.
+
+Introduction
+-------------
+
+Through this exercise, you will integrate NVIDIA FLARE with the popular
+deep learning framework `PyTorch <https://pytorch.org/>`_ and learn how to use NVIDIA FLARE to train a convolutional
+network with the CIFAR10 dataset using the included :class:`FedAvg<nvflare.app_common.workflows.fedavg.FedAvg>` workflow.
+
+The setup of this exercise consists of one **server** and two **clients**.
+
+The following steps compose one cycle of weight updates, called a **round**:
+
+ #. Clients are responsible for generating individual weight-updates for the model using their own CIFAR10 dataset. 
+ #. These updates are then sent to the server which will aggregate them to produce a model with new weights. 
+ #. Finally, the server sends this updated version of the model back to each client.
+
+For this exercise, we will be working with the ``hello-pt`` application in the examples folder.
+
+Let's get started. First clone the repo:
+
+.. code-block:: shell
+
+  $ git clone https://github.com/NVIDIA/NVFlare.git
+
+Remember to activate your NVIDIA FLARE Python virtual environment from the installation guide.
+
+Since you will use PyTorch and torchvision for this exercise, let's go ahead and install both libraries: 
+
+.. code-block:: shell
+
+  (nvflare-env) $ python3 -m pip install torch torchvision
+
+
+.. note::
+
+  There is a pending fix related to Pillow, PyTorch==1.9 and Numpy.  If you see exception related to
```

Comment:
Do we know What is this issue?

Reply:
This was copied over from the previous example, I tried to look into it briefly but did not find good specifics on this, perhaps don't need this anymore?

Reply:
@SYangster @IsaacYangSLA @holgerroth do you remember anything regarding this?

Reply:
ok, I see this traces back to long time ago, maybe we just remove these sections.

Reply:
removed